charts/spring-cloud-config-server/values.yaml

```diff
@@ -58,5 +58,5 @@ config:
   gitSearchpath: "{application}"
 
 secrets:
-  gitUsername:
-  gitPassword:
+  gitUsername: test
+  gitPassword: test
```

Comment:
imho there should be no default password. instead use random password when no password is set:

```
  {{- if Values.secrets.gitPassword }}
  gitPassword: {{ Values.secrets.gitPassword | b64enc | quote }}
  {{- else }}
  gitPassword: {{ randAlphaNum 40 | b64enc | quote }}
  {{- end }}
```

Reply:
Made the suggested changes ,Thank you